go.mod

```diff
@@ -0,0 +1,6 @@
+module github.com/scylladb/gocqlx
+
+require (
+	github.com/gocql/gocql v0.0.0-20180530083731-3c37daec2f4d
+	github.com/google/go-cmp v0.2.0
```

Comment:
This is a test requirement do we have some distinction for that in vgo?

Reply:
Afaik no. I think it works the same way as today. The test deps are just ignored bu the build commands and just used when actually running tests.

Reply:
If they are not listed here the tests won't work however so I guess they are needed in the mod file.